### PR TITLE
selinux role:  Changed Reboot process

### PR DESCRIPTION
**Pull request by Fishy78**
- Replaced the reboot process with the ansible reboot process
- Changed the selinux.changed condition to selinux.reboot_required

### CODE REVIEW

File: roles/selinux/tasks/main.yml [hunk 1]
```diff
@@ -18,18 +18,9 @@
...
-      become: false
-      delegate_to: localhost
-      wait_for:
-        port: "{{ ssh_port }}"
-        host: "{{ item.address }}"
-        search_regex: SSH
-        delay: 10
-  when: selinux.changed and reboot
+- name: SELinux | Restart machine
+  reboot:
```
Comment (ericsysmin) [inline roles/selinux/tasks/main.yml]:
```suggestion
  ansible.builtin.reboot:
```

Comment (ericsysmin) [reply roles/selinux/tasks/main.yml]: We also need to note that with the change it's only supported in 2.7 or later. Which is fine. I will include this in the new release I'm working on.

**Comment by ericsysmin** [comment]
@Fishy78 can you make the suggested changes?

**Review by ericsysmin — CHANGES_REQUESTED** [review]

**Review by ericsysmin — APPROVED** [review]

**Comment by Fishy78** [comment]
Sorry, you were too quick ;)

**Pull request merged**